src/Sarif.UnitTests/Writers/SarifLoggerTests.cs

```diff
@@ -34,24 +33,38 @@ public void SarifLogger_RedactedCommandLine()
             // "C:\Program Files (x86\\Microsoft Visual Studio 14.0\Common7\IDE\QTAgent32_40.exe\" 
             // /agentKey a144e450-ac06-46d0-8365-c21ea7872d23 /hostProcessId 8024 /hostIpcPortName 
             // eqt -60284c64-6bc1-3ecc-fb5f-a484bb1a2475"
+            // 
```

Comment:
While you're in here, could you fix the nit in `SarifLogger.CreateRun` where it redacts `run.Invocation.CommandLine` twice (on line 56 and 59 of SarifLogger.cs)?